packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -40,13 +40,9 @@ ouia: true
 
 Note: Table lives in its own package at [@patternfly/react-table](https://www.npmjs.com/package/@patternfly/react-table)!
 
-PatternFly has two implementations of a React table.
+The `TableComposable` component takes an explicit and declarative approach, and its implementation closely mirrors that of an HTML table.
 
-The first is the newer `TableComposable` component. It takes a more explicit and declarative approach, and its implementation more closely mirrors that of an HTML table. Generally, updates and new feature requests are implemented in the `TableComposable`.
-
-The second is the original `Table` component. It is configuration based and takes a less declarative and more implicit approach about laying out the table structure, such as the rows and cells within it. The documentation for the older table implementation can be found under the [React legacy](/components/table/react-legacy) tab.
-
-**For most common use cases, we recommend using `TableComposable`. Both implementations are supported and fully maintained.**
+The documentation for the deprecated legacy table implementation can be found under the [React deprecated](/components/table/react-deprecated) tab. It is configuration based and takes a less declarative and more implicit approach to laying out the table structure, such as the rows and cells within it.
```

Comment:
Would it also make sense to remove "legacy" from this sentence?

```suggestion
The documentation for the deprecated table implementation can be found under the [React deprecated](/components/table/react-deprecated) tab. It is configuration based and takes a less declarative and more implicit approach to laying out the table structure, such as the rows and cells within it.
```

Possibly a similar question for the "react deprecated" tab's heading of "Legacy Table" as to whether that should be updated to "Deprecated Table".

Reply:
Removed "legacy"!

As far as the heading, ultimately it looks like it makes the most sense to just remove it from both tabs. When TableComposable is renamed to Table, it won't make much sense to have another "Table" heading nested within the higher-level "Table" heading at the top of the page. 

Also since we are communicating in multiple places (the tab, the description, etc.) that the legacy variant is deprecated, we shouldn't need to specify "Deprecated table" again in the heading. Thank you for calling this out - nice catch 🙂 WDYT @thatblindgeye ?